### DISCUSSION TITLE
Czech phrases to support in HA. / Návrhy na fráze v češtině.

### DISCUSSION THREAD

**Discussion by schizza** [OHF-Voice/intents#500] [Supported Sentences]
Pište návrhy na fráze k hlasovému ovládání, které chybí nebo vás napadají.

**Comment by mikilbc** [comment]
> Pište návrhy na fráze k hlasovému ovládání, které chybí nebo vás napadají.

Ahoj, šlo by dát příklad jak to psát, aby to dávalo smysl pro nějaké zpracování? 
Jako například "Rožni světlo v kuchyni." nebo "Rožni světlo <>."

**Reply by schizza** [reply]
Moc nechápu dotaz. Myslíš, jak dát návrhy na povely?
Pokud bys chtěl ovládat světlo příkazem rožni, pak jen uvést, že to chceš přidat.
Nicméně zrovna příkaz rožni už je přidaný, viz. [sentences/cs/_common.yaml`](https://github.com/home-assistant/intents/blob/38df4fd79baca9f2370ebc865dfbecc13db008ed/sentences/cs/_common.yaml).

**Reply by mikilbc** [reply]
Díky, zaregistroval jsem na FB jen příspěvek přidávejte fráze a toto vlákno. 
Takhle s "sentences/cs/_common.yaml" to hned dává smysl. Díky

**Reply by Stooovie** [reply]
Z nějakého důvodu ale "Rožni" nedělá nic, ať už specifikuju jak chci. "Rožni světlo kuchyň" by měl podle light_hassturnon.yaml být legit požadavek a area je nastavená správně, ale vždy dostanu jen "při požadavku došlo k chybě"

Edit: pardon, ono to nefunguje ani v angličtině.

**Reply by schizza** [reply]
@Stooovie mě to funguje :)
![CleanShot 2023-02-02 at 10 56 20](https://user-images.githubusercontent.com/4604900/216293146-e35bebe7-7990-48fa-8ce5-acda14d4fb2b.gif)

**Reply by schizza** [reply]
@Stooovie a ještě edit. Přidáním aliasu to můžeš ještě počeštit :)
![CleanShot 2023-02-02 at 11 05 53](https://user-images.githubusercontent.com/4604900/216295437-14d256ad-8927-4aaa-87f0-aac5de37df36.gif)

**Reply by berkavil** [reply]
> Moc nechápu dotaz. Myslíš, jak dát návrhy na povely? Pokud bys chtěl ovládat světlo příkazem rožni, pak jen uvést, že to chceš přidat. Nicméně zrovna příkaz rožni už je přidaný, viz. [sentences/cs/_common.yaml`](https://github.com/home-assistant/intents/blob/38df4fd79baca9f2370ebc865dfbecc13db008ed/sentences/cs/_common.yaml).

Ahoj wespolek, Koukal jsem na yaml a skloňování (ohýbání) slov v CZ - super!
Jsem fakt moc zvědavej jak to bude fungovat až to v komunitě za pár týdnů odladíme.,. :-) 
Prosím - neviděl jsem v tom yamlu nikde příkazy na roztahování/zatahování rolet / žaluzií?  
Můžu někde editovat přímo ten yaml nebo se sem jen píšou návrhy a edituješ sám?
Díky.
Willy

**Reply by schizza** [reply]
> > Moc nechápu dotaz. Myslíš, jak dát návrhy na povely? Pokud bys chtěl ovládat světlo příkazem rožni, pak jen uvést, že to chceš přidat. Nicméně zrovna příkaz rožni už je přidaný, viz. [sentences/cs/_common.yaml`](https://github.com/home-assistant/intents/blob/38df4fd79baca9f2370ebc865dfbecc13db008ed/sentences/cs/_common.yaml).
> 
> 
> 
> Ahoj wespolek, Koukal jsem na yaml a skloňování (ohýbání) slov v CZ - super!
> 
> Jsem fakt moc zvědavej jak to bude fungovat až to v komunitě za pár týdnů odladíme.,. :-) 
> 
> Prosím - neviděl jsem v tom yamlu nikde příkazy na roztahování/zatahování rolet / žaluzií?  
> 
> Můžu někde editovat přímo ten yaml nebo se sem jen píšou návrhy a edituješ sám?
> 
> Díky.
> 
> Willy

Žaluzije jsou v homeassistant_HassTurnOn a TurnOff

**Reply by berkavil** [reply]
Ahoj, to asi nebude ono, mám na mysli předokenní rolety - ty jsou v šabloně "covers" a příkaz snad není on/off, ale open/close a dá se udat i konkrétní procentuální poloha. Toto by v CZ bylo něco jako: "zatáhnout rolety 30%" ?
možná se pletu? dík.

EDIT: v tom CS slovníku vidím otevřít/zavřít a nastavit... tak by se covers ovládaly asi jedním z těchto 3 příkazů? (roztáhnout/zatáhnout pomocí otevřít/zavřít a ta poloha pomocí nastavit- např. "rolety nastavit 30%" ?)
Dík.
W.

**Reply by schizza** [reply]
> Ahoj, to asi nebude ono, mám na mysli předokenní rolety - ty jsou v šabloně "covers" a příkaz snad není on/off, ale open/close a dá se udat i konkrétní procentuální poloha. Toto by v CZ bylo něco jako: "zatáhnout rolety 30%" ? možná se pletu? dík.
> 
> EDIT: v tom CS slovníku vidím otevřít/zavřít a nastavit... tak by se covers ovládaly asi jedním z těchto 3 příkazů? (roztáhnout/zatáhnout pomocí otevřít/zavřít a ta poloha pomocí nastavit- např. "rolety nastavit 30%" ?) Dík. W.

v `_commons.yaml` jsou definovány příkazy `roztahnout: "(roz|vy|od)táhn(out|i)"`
a ve větách 

```
      - sentences:
          - "(<otevrit>|<roztahnout>) {name}"
        response: cover
      - sentences:
          - "(<otevrit>|<roztahnout>) {name} [v] {area}"
        response: cover_area
```

na přímou polohu zatím nejsou definovány pozice, zkusím, jestli to hassil zchroupne a uvidíme.

**Reply by berkavil** [reply]
> ... 
> na přímou polohu zatím nejsou definovány pozice, zkusím, jestli to hassil zchroupne a uvidíme.

Ok díky.
na set_position maj rolety (covers) službu - definice dle templatu https://www.home-assistant.io/integrations/cover.template/
Tj. když chci nastavit pozici, zavolá se skriptem/automatem služba, kde vstupní parametr je to procento... a ještě tušim že jde volitelný parametr "reverse", ten ale nepoužívám, když některý rolety maj jako default nastaveno 100% zavřeno, jiné to mají opačně.
W.

**Reply by schizza** [reply]
> > ...
> > na přímou polohu zatím nejsou definovány pozice, zkusím, jestli to hassil zchroupne a uvidíme.
> 
> Ok díky. na set_position maj rolety (covers) službu - definice dle templatu https://www.home-assistant.io/integrations/cover.template/ Tj. když chci nastavit pozici, zavolá se skriptem/automatem služba, kde vstupní parametr je to procento... a ještě tušim že jde volitelný parametr "reverse", ten ale nepoužívám, když některý rolety maj jako default nastaveno 100% zavřeno, jiné to mají opačně. W.

Problém je v tom, že pokud to nebude zakomponováno přímo v hassil knihovně, tak to nevezme z HA a nepředá stav rolety. Musí umět ze `sentences` přečíst `{ state }` a předat do `intents` požadovaný stav, tak jako je to třeba u světla u `brightness` nebo `color`. Ale jelikož ještě není ani definováno, respektive hází chyby v teplotě - `{{ state.state }}` hodí chybu na nedefinovanou hodnotu, tak se obávám, že procentuální stav u rolet taky nebude.
Upřímně, zatím toho nefunguje dost ani v podporovaných intents. :/

**Reply by berkavil** [reply]
OK, jasný, neva. Však se to celé jmenuje "the year of the voice :-)" ... a teď je teprve únor, ne?  :-)
Díky, že to někdo v CZ vůbec řeší.
Čus W.

**Comment by bigboban** [comment]
Trochu odbočím od tématu cz patternů ALE - tušíte někdo jaký HW vůbec půjde použít k tomu ovládání? Bylo by asi dobrý si to hned zkoušet co tu plodíme. Aby ten "year of voice" neskončil tím že HA bude mít "dokonalý" cz patterny bez možnosti to prakticky jakkoliv využít..

**Reply by berkavil** [reply]
> ... - tušíte někdo jaký HW vůbec půjde použít k tomu ovládání?
...
Ahoj - to mě taky napadlo - jak to vlastně bude fungovat po HW stránce? Zatím to (teda aspoň u mě) žádné mluvené slovo nerozpoznává... koukám se na HA v Chromu, z notebooku s mikrofonem "On"...  ale textové okno, zdá se, v CZ nějak komunikuje pomocí těch CZ hlášek.

**Reply by schizza** [reply]
Zatím jde použít Google a Siri, bohužel ani jeden pořádně nerozumí česky :/
Ale bude možnost i lokálního zpracování hlasu s tím, že by to mělo umět i CZ. Ale na to bude asi třeba silnější HW než RPI.
Zatím lze v Assistentovi psát příkazy, které by pak měly být pomocí STT zpracovaný.

jinak jak to funguje zde: [YouTube](https://www.youtube.com/watch?v=ixgNT3RETPg) a zde [HA](https://www.home-assistant.io/blog/2023/01/26/year-of-the-voice-chapter-1/#read-more)

**Reply by bigboban** [reply]
Google? Nechápu. Mám samozřejmě google mini ale přes něj tenhle koncept skoro určitě použít NEJDE. Podle mě je celkem škoda že tu školu stavěj jak v kocourkově od střechy. Řeší se tu SW stránka ale není zatím rozumná cesta jak ty komandy do HA "krmit"? Jako je hezký že si to pokusně můžu psát v browseru do HA frontendu.. Ale já samozřejmě potřebuju HW rozmístit po bytě a ideálně hezký HW ne nějaký hnusný bastl. Google home mini koupím za pětikilčo a přesně tak nějak si představuju že ovládací HW má vypadat..

**Reply by schizza** [reply]
> Google? Nechápu. Mám samozřejmě google mini ale přes něj tenhle koncept skoro určitě použít NEJDE. Podle mě je celkem škoda že tu školu stavěj jak v kocourkově od střechy. Řeší se tu SW stránka ale není zatím rozumná cesta jak ty komandy do HA "krmit"? Jako je hezký že si to pokusně můžu psát v browseru do HA frontendu.. Ale já samozřejmě potřebuju HW rozmístit po bytě a ideálně hezký HW ne nějaký hnusný bastl. Google home mini koupím za pětikilčo a přesně tak nějak si představuju že ovládací HW má vypadat..

Koukni na videa. Krmit je můžeš, ale bohužel ani Siri ani Google neumí česky. Jsou applety do hodinek i do Domácnosti u Apple, ale když ti Siri nerozomí, blbě se na ní mluví česky.

**Reply by berkavil** [reply]
> Google? Nechápu. Mám samozřejmě google mini ale přes něj tenhle koncept skoro určitě použít NEJDE.
> Podle mě je celkem škoda že tu školu stavěj jak v kocourkově od střechy. Řeší se tu SW stránka ale není zatím rozumná 
> cesta jak ty komandy do HA "krmit"? Jako je hezký že si to pokusně můžu psát v browseru do HA frontendu..

Pardon :-) ...
jak myslíš, že vznikala siri nebo alexa? Nejdřív někdo musel vyrobit SW základnu a sadu příkazů a otestovat to v nějakym podobnym frontendu, HW pak nasadíš a budeš testovat, až bude co testovat... jak jinak bys to chtěl dělat opačně??

**Reply by berkavil** [reply]
Předpokládám, že výsledně bude jedno, jakým HW to budeš chtít komandovat, To je právě na HA tak skvělý! Že to neni závislé na konkrétním výrobci najěkého HW, který tě pak nutí brát i s chlupama vše jen od něj a ideálně vše uzamčeno, abys neviděl kam komu co posílá za data... že, Apple? (i když dnes už i Google, Amazon, Huawei apod.... )
HA Rulez! :-)

**Reply by bigboban** [reply]
Ne podle mě postup jak rozchodit hlasový ovládání je:

1. mít HW
2. skrz HW umět nahrát zvuk 
3. z toho zvuku zjistit co je to za text, tedy převod hlasu na text
4. no a nad daným textem se snažit vykoumat co přesně ten komand znamená

To co je potřeba ladit je bod č. 3 a 4. U dvojky max nějaký technikálie jako eliminace šumu apod.

Trojku asi teď neřešíme a spoleháme se že technoogie gůglu to má vyřešený dokonale úplně stejně jako TTS (text -> hlas). Tedy aktuálně HA vyhlásil že řeší #4. 

No ALE - k čemu mi bude bod č. 4 bez alespoň nějak funkčního HW z bodu 1? Na teploměry, přehrávání zvuku apod máme tuny HW integrovatelného do HA. Myslím profi zařízení co si koupím na Alze NE bastl.

Ale dokáže dneska někdo uvést byť jediný příklad HW co mi tady k tomu poslouží? Buď mi něco uniká ale podle mě prostě neexistuje. A na konci roku jako existovat bude? Bylo by dobré mít aspoň představu jak to celý bude HW fungovat..

**Reply by schizza** [reply]
> Ne podle mě postup jak rozchodit hlasový ovládání je:
> 
> 1. mít HW
> 2. skrz HW umět nahrát zvuk
> 3. z toho zvuku zjistit co je to za text, tedy převod hlasu na text
> 4. no a nad daným textem se snažit vykoumat co přesně ten komand znamená
> 
> To co je potřeba ladit je bod č. 3 a 4. U dvojky max nějaký technikálie jako eliminace šumu apod.
> 
> Trojku asi teď neřešíme a spoleháme se že technoogie gůglu to má vyřešený dokonale úplně stejně jako TTS (text -> hlas). Tedy aktuálně HA vyhlásil že řeší #4.
> 
> No ALE - k čemu mi bude bod č. 4 bez alespoň nějak funkčního HW z bodu 1? Na teploměry, přehrávání zvuku apod máme tuny HW integrovatelného do HA. Myslím profi zařízení co si koupím na Alze NE bastl.
> 
> Ale dokáže dneska někdo uvést byť jediný příklad HW co mi tady k tomu poslouží? Buď mi něco uniká ale podle mě prostě neexistuje. A na konci roku jako existovat bude? Bylo by dobré mít aspoň představu jak to celý bude HW fungovat..


Koukni na stream YT a uvidíš, že základní řešení hlasového ovládání je hotové. Teď neřeší jaký HW, protože to funguje. Teď se řeší aby fungovaly commandy, tak jak mají. A samozřejmě, pak se bude doplňovat podpora různých HW řešení.
Viz třeba Rhasspi.
STT funguje, problém je, že velký hráči - Google, Apple nerozumí česky :/

**Comment by pepppek** [comment]
ahoj, prubezne testuju (pres shortcut button v telefonu) - zapnout / vypnout svetla funguje ok. problem je u zmeny barvy. nerozumi slovu změň - při použití slova "změnit" funguje ok. barvy jdou ok.
Zatim jsem ještě nepřišel na to, jestli se dá nějakým způsobem spustit script nebo automatizaci. Existuje neco jako alias pro skritp? - aby se dalo řict pouze zapni "XXX" ?

**Reply by schizza** [reply]
Světlo by mělo jít ovládat těmito příkazy? `  rozsvitit: "(rozsviť|rozsvítit|rožn[i|out])" #some dialect to be served for users :)
  zhasnout: "(zhasni|zhasnout)"`
změna barvy `zmenit: "(změň|změnit)"`

**Comment by maceace** [comment]
Zdravím,
nemělo by být na těchto místech místo "area" spíš "{name}"?
 
![ton](https://user-images.githubusercontent.com/66499150/217051787-41f5b016-b62f-4703-bbba-ad682eb9bd12.PNG)
![toff](https://user-images.githubusercontent.com/66499150/217051789-08e9ac30-0b57-4380-b9df-be3e94786bb6.PNG)

**Reply by schizza** [reply]
Mají tam být obě možnosti, ale .... #770

**Reply by maceace** [reply]
já jsem prostě přesvědčen že
"Rozsviť světlo" by mělo cílit na {name}
a
"Rozsviť světla" na {area}
Protože "Rozsviť světlo v kuchyni." je rozdíl než "Rozsviť světla v kuchyni.".

Nebo tedy proč když řeknu " Vypni světlo (jednotné číslo) v kuchyni" mi zhasnou všechna světla v kuchyni .

Dokud toto nebude změněno, není ani možnost individuální opravy, pokud tedy nedám světlům v kuchyni aliasy "svetlo Petr" a druhému "světlo Pavel".

**Reply by schizza** [reply]
> já jsem prostě přesvědčen že "Rozsviť světlo" by mělo cílit na {name} a "Rozsviť světla" na {area} Protože "Rozsviť světlo v kuchyni." je rozdíl než "Rozsviť světla v kuchyni.".
> 
> Nebo tedy proč když řeknu " Vypni světlo (jednotné číslo) v kuchyni" mi zhasnou všechna světla v kuchyni .
> 
> Dokud toto nebude změněno, není ani možnost individuální opravy, pokud tedy nedám světlům v kuchyni aliasy "svetlo Petr" a druhému "světlo Pavel".

Ano, rozsviť světlo by mělo cílit na {name}, ale generuje to zatím chybu - vyžaduje `domain: light`, ale `domain: light` nelze přidat jedné entitě. Takže, zatím to je takto, dokud se neopraví bug. 
Víme o tom.

**Reply by schizza** [reply]
A je zajímavé, že to dělá jen u lights_HassTurnOn/Off

**Reply by maceace** [reply]
dobrá hloupost je také: "ztlumit jas na maximum"

**Reply by schizza** [reply]
> dobrá hloupost je také: "ztlumit jas na maximum"

jj, díky.  #938

**Reply by maceace** [reply]
> > já jsem prostě přesvědčen že "Rozsviť světlo" by mělo cílit na {name} a "Rozsviť světla" na {area} Protože "Rozsviť světlo v kuchyni." je rozdíl než "Rozsviť světla v kuchyni.".
> > Nebo tedy proč když řeknu " Vypni světlo (jednotné číslo) v kuchyni" mi zhasnou všechna světla v kuchyni .
> > Dokud toto nebude změněno, není ani možnost individuální opravy, pokud tedy nedám světlům v kuchyni aliasy "svetlo Petr" a druhému "světlo Pavel".
> 
> Ano, rozsviť světlo by mělo cílit na {name}, ale generuje to zatím chybu - vyžaduje `domain: light`, ale `domain: light` nelze přidat jedné entitě. Takže, zatím to je takto, dokud se neopraví bug. Víme o tom.

No ale pokud by se:
![IMG_20230207_140511](https://user-images.githubusercontent.com/66499150/217253454-2385a346-654f-4582-974e-f4389fe03869.jpg)
![IMG_20230207_140641](https://user-images.githubusercontent.com/66499150/217253501-fa328c32-b235-4bc9-aea2-8977336c1196.jpg)
vyhodilo z light_HassTurnOn.yaml a light_HassTurnOff.yaml
a vložilo do homeassistant_HassTurnOn.yaml a homeassistant_HassTurnOff.yaml
s přepsáním z area na {name}
tak by to mělo být funkční.

**Reply by schizza** [reply]
> > > já jsem prostě přesvědčen že "Rozsviť světlo" by mělo cílit na {name} a "Rozsviť světla" na {area} Protože "Rozsviť světlo v kuchyni." je rozdíl než "Rozsviť světla v kuchyni.".
> > > Nebo tedy proč když řeknu " Vypni světlo (jednotné číslo) v kuchyni" mi zhasnou všechna světla v kuchyni .
> > > Dokud toto nebude změněno, není ani možnost individuální opravy, pokud tedy nedám světlům v kuchyni aliasy "svetlo Petr" a druhému "světlo Pavel".
> > 
> > 
> > Ano, rozsviť světlo by mělo cílit na {name}, ale generuje to zatím chybu - vyžaduje `domain: light`, ale `domain: light` nelze přidat jedné entitě. Takže, zatím to je takto, dokud se neopraví bug. Víme o tom.
> 
> No ale pokud by se: ![IMG_20230207_140511](https://user-images.githubusercontent.com/66499150/217253454-2385a346-654f-4582-974e-f4389fe03869.jpg) ![IMG_20230207_140641](https://user-images.githubusercontent.com/66499150/217253501-fa328c32-b235-4bc9-aea2-8977336c1196.jpg) vyhodilo z light_HassTurnOn.yaml a light_HassTurnOff.yaml a vložilo do homeassistant_HassTurnOn.yaml a homeassistant_HassTurnOff.yaml s přepsáním z area na {name} tak by to mělo být funkční.

Nelze ale přesunou doménu light do homeassistant. Jsou oddělené záměrně. světla se ovládají z light_hassTurnOn/Off, ostatní přes homeassistant_TurnOn/Off.

**Reply by maceace** [reply]
Omlouvám se za svoji laickou selskou logiku.
Pokud po odstranění chyby bude Assist umět "Rozsvítit/Zhasnout (konkrétní) světlo {name}", je následující bezpředmětné:
Nemyslel jsem přesouvat doménu, ale pokud může homeassistant_HassTurnOn/off  "Zapnout/Vypnout {name} ((světlo))" mohl by ho i "Rozsvítit/Zhasnout ((světlo)) {name}.
Slovo ((světlo)) v těch větách by tu doménu zastoupilo.
Nyní je prostě matoucí větou "Zhasni světlo kuchyň" zhasnout všechna světla v kuchyni.

**Reply by schizza** [reply]
Fix #946

**Reply by schizza** [reply]
Takže v HA nightly by mělo být už vyřešeno.
Je to sice řešení přes sílu, ale zatím funguje.

**Reply by ra-kal** [reply]
Ahoj. Rozlišoval bych mezi "rozsvítit" a "zapnout". Zatímco u rozsvítit je jednoznačné, že se jedná o světlo, zapnout můžu chtít třeba ventilátor. To samé pro vypnout/zhasnout
Příklad:

- Rozsviť nad stolem
- Zapni světlo nad stolem
- Zapni ventilátor nad stolem

Dále bych u oblastí povolil ve větě "v"

- "(<zapnout>|<rozsvitit>) [všechna] [světla] [v] <area>"
- "[v] <area> (<zapnout>|<rozsvitit>) [všechna] světla"

Příklad:
Oblast mám nazvanou Koupelna s aliasem "koupelně". Rozsviť/zhasni v koupelně ale nyní nefunguje.

**Reply by schizza** [reply]
> Ahoj. Rozlišoval bych mezi "rozsvítit" a "zapnout". Zatímco u rozsvítit je jednoznačné, že se jedná o světlo, zapnout můžu chtít třeba ventilátor. To samé pro vypnout/zhasnout Příklad:
> 
> * Rozsviť nad stolem
> * Zapni světlo nad stolem
> * Zapni ventilátor nad stolem
> 
> Dále bych u oblastí povolil ve větě "v"
> 
> * "(|) [všechna] [světla] [v] "
> * "[v]  (|) [všechna] světla"
> 
> Příklad: Oblast mám nazvanou Koupelna s aliasem "koupelně". Rozsviť/zhasni v koupelně ale nyní nefunguje.

@ra-kal 
Kouknu na to, ale tohle by mělo fungovat :/

**Comment by maceace** [comment]
zde jsem si dovolil zkombinovat všechna použitá slova:
https://bit.ly/assist_cz_fraze

**Comment by ondrasy** [comment]
Ahoj, když mám alias "na půdě" a napíšu "rozsviť na půdě", tak to správně rozsvítí všechna světla na půdě. Zhasnout bych je chtěl podobně: "zhasni na půdě", ale to nefunguje, protože tam chce ještě slovo světla, což si myslím, že není potřeba... Můžete se na to, prosím, podívat? Díky

**Reply by sairon** [reply]
Opraveno v #1703.

**Comment by PetrMa** [comment]
Ahoj všem,

trochu bojuju s těmi českými větami.

Např, zjištění teploty v oblasti, která se jmenuje Pokoj.
Když se zeptám podle vzoru "jak teplo je pokoj" přičemž v oblasti Pokoj mám jediné zařízení které měří teplotu, vždy dostávám zpátky odpověď "Omlouvám se, ale nerozumím". V podstatě stejnou odpověď dostávám i na spoustu jiných příkazů. To samé dostávám i na dotaz "jaká je pokoj teplota" atd atd.

Debug odpovědi je (nicméně z toh nedokážu vyčíst kde je chyba):
init_options:
  start_stage: intent
  end_stage: intent
  input:
    text: jak teplo je pokoj
  pipeline: 01gzjpty92xt57ps24nhtpzay9
  conversation_id: null
stage: done
run:
  pipeline: 01gzjpty92xt57ps24nhtpzay9
  language: cs
  runner_data:
    stt_binary_handler_id: null
    timeout: 300
events:
  - type: run-start
    data:
      pipeline: 01gzjpty92xt57ps24nhtpzay9
      language: cs
      runner_data:
        stt_binary_handler_id: null
        timeout: 300
    timestamp: "2023-11-07T11:51:47.586221+00:00"
  - type: intent-start
    data:
      engine: homeassistant
      language: cs
      intent_input: jak teplo je pokoj
      conversation_id: null
      device_id: null
    timestamp: "2023-11-07T11:51:47.586381+00:00"
  - type: intent-end
    data:
      intent_output:
        response:
          speech:
            plain:
              speech: Omlouvám se, ale nerozumím
              extra_data: null
          card: {}
          language: cs
          response_type: error
          data:
            code: no_intent_match
        conversation_id: null
    timestamp: "2023-11-07T11:51:47.607426+00:00"
  - type: run-end
    data: null
    timestamp: "2023-11-07T11:51:47.607597+00:00"
intent:
  engine: homeassistant
  language: cs
  intent_input: jak teplo je pokoj
  conversation_id: null
  device_id: null
  done: true
  intent_output:
    response:
      speech:
        plain:
          speech: Omlouvám se, ale nerozumím
          extra_data: null
      card: {}
      language: cs
      response_type: error
      data:
        code: no_intent_match
    conversation_id: null

Neřešil to už někdo? Co dělám blbě? díky

**Reply by sairon** [reply]
V českých intents je pouze [climate_HassClimateGetTemperature](https://github.com/home-assistant/intents/blob/main/sentences/cs/climate_HassClimateGetTemperature.yaml). Jak název napovídá, týká se pouze entit typu `climate`, což zřejmě Tvůj teploměr není. Aby fungovaly i všeobecné senzory, bude potřeba udělat českou verzi vět `sensor_HassGetState`, které byly i do anglické mutace přidány v září: https://github.com/home-assistant/intents/blob/main/sentences/en/sensor_HassGetState.yaml

**Reply by PetrMa** [reply]
Já se ale neptám teploměru, ale hlavice, takže ta jako climate je v dané místnosti. I přesto to ale nereaguje.
